.travis.yml

```diff
@@ -1,12 +1,9 @@
 language: node_js
 node_js:
-  - 4    # latest 4.x release
```

Comment:
maybe keep 4 around if we have lagging users of this client

Reply:
:+1: Lets just hope the tests passes :)